src/component/variantPage/functionalPrediction/AlphaMissense.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import { makeObservable, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import functionalGroupsStyle from '../functionalGroups.module.scss';
+
+export interface IAlphaMissenseProps {
+    amClass?: string;
+    amPathogenicityScore?: number;
+}
+
+const ALPHAMISSENSE_URL = 'https://www.science.org/doi/10.1126/science.adg7492';
+
+const AlphaMissenseInfo: React.FunctionComponent = () => {
+    return (
+        <div>
+            <a
+                href={ALPHAMISSENSE_URL}
+                target="_blank"
+                rel="noopener noreferrer"
+            >
+                AlphaMissense
+            </a>{' '}
+            uses deep learning to assess the pathogenicity of missense variants,
+            indicating how they might affect gene function and contribute to
+            disease.
+            <div>
+                <b>Scores:</b>
+                <ul>
+                    <li>
+                        <b>Pathogenic:</b> Score &gt; 0.564 – likely to cause
+                        disease.
+                    </li>
+                    <li>
+                        <b>Benign:</b> Score &lt; 0.34 – unlikely to cause
+                        disease.
+                    </li>
+                    <li>
+                        <b>Ambiguous:</b> Score between 0.34 and 0.564 – unclear
+                        impact; insufficient data.
+                    </li>
+                </ul>
+            </div>
+        </div>
+    );
+};
+
+@observer
+export default class AlphaMissense extends React.Component<
+    IAlphaMissenseProps,
+    {}
+> {
+    @observable showDetails = false;
+
+    constructor(props: IAlphaMissenseProps) {
+        super(props);
+        makeObservable(this);
+    }
+
+    public render() {
+        let alphaMissenseContent: JSX.Element;
+
+        const dataSource = (
+            <>
+                AlphaMissense&nbsp;
+                <i className="fas fa-external-link-alt" />
+            </>
+        );
+        if (
+            this.props.amClass &&
+            this.props.amClass.length > 0 &&
+            this.props.amClass !== 'N/A'
```

Comment:
Is it even possible to have 'N/A'?

Reply:
And do we need to check both this.props.amClass && this.props.amClass.length > 0?

Reply:
amClass has a default value of 'na'. The checks are in place to ensure that the value is valid, not an empty string, and not equal to 'N/A', which helps prevent any issues with uninitialized or default values.

Reply:
@JunheZoooo 
```
const amClass =
            genomeNexusData?.annotation_summary?.transcriptConsequenceSummary
                ?.alphaMissense?.pathogenicity || undefined;
```
Isn't the default value `undefined`?

Reply:
changed it